src/NUnitTestAdapter/Internal/Extensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace NUnit.VisualStudio.TestAdapter.Internal
+{
+#if NET35
+
+    public static class TypeExtensions
+    {
+        public static Type GetTypeInfo(this Type type) => type;
+    }
+#endif
+    public static class StringExtensions
+    {
+        public static bool IsNullOrWhiteSpace(this string value)
+        {
+            return value == null || value.Trim().Length == 0;
```

Comment:
It's good to stay simple since it's not in a hot path. 👍

For comparison: https://referencesource.microsoft.com/#mscorlib/system/string.cs,55e241b6143365ef

Reply:
Yes, just used the https://msdn.microsoft.com/en-us/library/system.string.isnullorwhitespace(v=vs.110).aspx as a reference, and under remarks this code is shown as equivalent, and it is easier than the code from the referencesource.